bundle/compliance/cis_gcp/rules/cis_1_12/rule.rego

```diff
@@ -0,0 +1,20 @@
+package compliance.cis_gcp.rules.cis_1_12
+
+import data.compliance.lib.common
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.if
+
+default is_valid = false
+
+is_valid if {
+	not startswith(data_adapter.resource.data.name, "projects/")
```

Comment:
according to [gcp cloud-platform-resource-hierarchy](https://cloud.google.com/resource-manager/docs/cloud-platform-resource-hierarchy):
1 - projects start with `projects/`
2 - folders start with `folders/`
3 - orgs starts with `organizations/`